<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="output_path_label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PATH  :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="output_path_line_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="image_format_groupbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Image Format</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QCheckBox" name="png_checkbox">
          <property name="text">
           <string>png</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="jpg_checkbox">
          <property name="text">
           <string>jpg</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="gif_checkbox">
          <property name="text">
           <string>gif</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="svg_checkbox">
          <property name="text">
           <string>svg</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="edit_button">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
